apps/web/src/components/HoverCard/HoverCardOnUsername.tsx

```diff
@@ -157,7 +157,7 @@ const StyledCardContainer = styled.div`
   width: 375px;
   min-height: 128px;
   max-width: calc(100vw - ${pageGutter.mobile * 2}px);
-  display: grid;
```

Comment:
we also have a corresponding component for communities, `HoverCardOnCommunity`. should we update this there as well?